### PR TITLE
books: add skeleton and docs

### PR DESCRIPTION
**Pull request by hinto-janai**
- Adds `README.md` files for `books/`
- Adds `books/` related info in `CONTRIBUTING.md`
- Adds book build files to `.gitignore`
- Adds docs/books to LICENSE (MIT)

### CODE REVIEW

**Comment by hinto-janai** [comment]
@Boog900
- Should I move `monero-book`'s source here, now?
- I'll going to keep `architecture-book` in a separate repo for now since things will change fast and there's stuff to figure out
- Can you add a DNS entry for `architecture.cuprate.org` pointing to https://github.com/Cuprate/architecture-book?

**Comment by Boog900** [comment]
> Should I move monero-book's source here, now?

I'll do this in a separate PR 

> Can you add a DNS entry for architecture.cuprate.org pointing to https://github.com/Cuprate/architecture-book?

done, I think you will have to deploy something there to see if it worked

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -64,3 +64,8 @@ This section is primarily targeted at maintainers. Most contributors aren't able
 [I-]: https://github.com/Cuprate/cuprate/labels?q=I
 [O-]: https://github.com/Cuprate/cuprate/labels?q=O
 [P-]: https://github.com/Cuprate/cuprate/labels?q=P
+
+## Books
+Cuprate has various documentation books whose source files live in [`books/`](https://github.com/Cuprate/cuprate/tree/main/books).
+
+Please contribute if you found a mistake! The files are mostly [markdown](https://wikipedia.org/wiki/Markdown) files and can be easily edited. See the `books/` directory for more information.
```
Comment (Boog900) [inline CONTRIBUTING.md:71]:
```suggestion
Please contribute if you find a mistake! The files are mostly [markdown](https://wikipedia.org/wiki/Markdown) files and can be easily edited. See the `books/` directory for more information.
```

**Review by Boog900 — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (Boog900) [reply CONTRIBUTING.md:71]: wait I think both ok, my bad

**Review by Boog900 — APPROVED** [review]

**Pull request merged**